docs/whats-new-in-3.0.md

```diff
@@ -67,6 +67,24 @@ Consequently the following configuration options have been removed.
 
 For discussion and pull request see issues [#105](https://github.com/wsdl2phpgenerator/wsdl2phpgenerator/issues/105) and [#154](https://github.com/wsdl2phpgenerator/wsdl2phpgenerator/issues/154).
 
+### Configuring the stream context ###
+
+You can specify the stream context options using the `streamContextOptions` configuration parameter. This is especially useful for SSL configurations, such as ciphers, or local certificate validation, etc:
+ 
+```php
+new Wsdl2PhpGenerator\Config([
+    /* ... */
+    'streamContextOptions' => [
+        'ssl' => [
+            /* SSL specific context options here */
+        ]
+    ]
+    /* ... */
+])
+```
+
+Note that if you also specify the `stream_context` option as part of the `soapClientOptions`, you will get an exception. 
```

Comment:
As I see it this will break backwards compatibility. That may be OK since we are planning a new major release, 4.0.0, anyway, where this could go in.